templates/base.html

```diff
@@ -8,8 +8,13 @@
 		<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap-theme.min.css">
 		<script src="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
 		<link rel="stylesheet" href="{% static 'css/style.css' %}">
+		<link rel="stylesheet" href="{% static 'css/footer.css' %}">
```

Comment:
@anuschka no need for additional .css file. Everything can be in one having multiple files is not optimal as the client makes request for everyone separately.